script/munin-httpd

```diff
@@ -24,9 +24,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 =cut
 
-# Trust PERL5LIB from environment
```

Comment:
I cannot really comment on why these lines where here, but could you explain, why they should go away?

Reply:
`PERL5LIB` is a trick akin to `LD_LIBRARY_PATH`, but for Perl. 

This should not be used in a daemon listening on a TCP port as UID=0.